src/app/main/time-groups/edit-time-group/edit-time-group.component.html

```diff
@@ -206,8 +242,22 @@ <h1 class="size-small ml-1">{{employee.name}}</h1>
                 </tr>
               </tbody>
             </table>
+            <div class="text-center flex flex-middle flex-center"
+            *ngIf="!timeGroupLoading && filteredTimeGroupEmployees?.length ==0">
+            <div class="ptb-2 plr-2">
+              <img src="assets/images/placeholders/zones.svg" width="80" alt="">
```

Comment:
Static width is working for the responsive view?

Reply:
I tried it and it's working for all sizes